tutorials/polkadot-sdk/build-a-parachain/prepare-relay-chain.md

```diff
@@ -0,0 +1,210 @@
+
+The sample chain specification is only valid for a single parachain with two validator nodes. If you add other validators, add additional parachains to your relay chain, or want to use custom account keys instead of the predefined account, you'll need to create a custom chain specification file.
+
+Suppose you are completing this tutorial simultaneously as anyone on the same local network. In that case, you must download and modify the plain sample relay chain spec to prevent accidentally peering with their nodes. Find the following line in the plain chain spec and add characters to make the `protocolId` field unique:
+
+```json
+"protocolId": "dot",
+```
+
+## Start the Relay Chain Node
+
+Before starting block production for a parachain, you need to start a relay chain for them to connect.
+
+To start the validator nodes, follow these steps:
+
+1. Generate the chain specification file in the plain text format and use it to create the raw chain specification file. Save the raw chain specification file in a local working directory.
+
+    1. The plain text chain specification file can be generated by running the following command:
+
+        ```bash
+        ./target/release/polkadot build-spec --chain rococo-local-testnet > /tmp/plain-local-chainspec.json
+        ```
+
+    2. Convert the plain text chain specification file to the raw format by running the following command:
+
+        ```bash
+        ./target/release/polkadot build-spec --chain plain-local-chainspec.json --raw > /tmp/raw-local-chainspec.json
+        ```
+
+2. Start the first validator using the alice account by running the following command:
+
+      ```bash
+      ./target/release/polkadot \
+      --alice \
+      --validator \
+      --base-path /tmp/alice \
+      --chain /tmp/raw-local-chainspec.json \
+      --port 30333 \
+      --rpc-port 9944 \
+      --insecure-validator-i-know-what-i-do \
+      --force-authoring
+      ```
+
+    This command uses `/tmp/raw-local-chainspec.json` as the location of the sample chain specification file. Be sure the `--chain` command line specifies the path to the raw chain specification you downloaded into a local working directory. This command also uses the default values for the port (`port`) and WebSocket port (`ws-port`). The values are explicitly included here as a reminder to always check these settings. After the node starts, no other nodes on the same local machine can use these ports.
+
+3. Review log messages as the node starts and take note of the `Local node identity` value. This value is the node’s peer ID, which you need to connect the parachain to the relay chain
+
+      <div id="termynal" data-termynal>
```

Comment:
I wouldn't mind if the command is not repeated here as you explained it a few lines above